### DISCUSSION TITLE
how to type quotes?

### DISCUSSION THREAD

**Discussion by maxandersen** [charmbracelet/vhs#141] [Q&A] [answered]
How does one get "Type" to type quotes?

I tired various combinations but none seem to work:

```
Type "jbang -c 'println(\"this is java\");'"
```

but all either get syntax error or the quote is simply not included.

Any pointers would be great!

**Comment by caarlos0** [comment, marked as answer]
I think you could:

```elixir
Type "jbang -c 'println"
Type '("this is java")'
Type ";'"
```

although, not escaping the quotes is likely something we need to improve... cc/ @maaslalani

**Reply by maaslalani** [reply]
Yep, I think I can add something like:
```
Type `jbang -c 'println("this is java");'`
```

for these kinds of scenarios?

**Reply by caarlos0** [reply]
sounds good yes!

**Comment by maaslalani** [comment]
@maxandersen BTW, you can now (on VHS `main`) use backticks to `Type` both single and double quotes in the same string:

```
Type `"hello" 'world'`
```
